versioned_docs/version-latest/protocol/pulsar.mdx

```diff
@@ -469,7 +404,111 @@ Initial plotting for plot: #0 (/home/username/.local/share/pulsar/farms)
 
 That's it! Enjoy and Happy Farming!
 
-#### Moving the Farming Process to the Background
+
+## Troubleshooting
+
+---
+
+### Required ports
+
+Currently, a few ports need to be exposed for node to work properly.
+
+If you have a server with no firewall, there is nothing to be done, but otherwise make sure to open the following TCP ports for incoming connections.
+
+- `30333`
+- `30433`
+- `30533`
+
+On the desktop side if you have a router in front of your computer, you'll need to forward TCP ports to the machine on which your node is running (how this is done varies from router to router, but there is always a feature like this, refer to [How to Forward Ports](port-forwarding.md) for a more in-depth tutorial).
+If you're connected directly without any router, then again nothing needs to be done in such case.
+
+
+### View your Logs
```

Comment:
While we're here, let's rename it something like **Inspecting logs**. Feel free to suggest some other name, but View your Logs sounds a bit out of place.

Reply:
Agreed, inspecting logs sounds better here.

Reply:
I kind of like Inspecting *Farmer* Logs? Hmm. If you think of something better we can die on this hill haha. These things are important 😄

Reply:
Definitely! 
I'd suggest to not use a word "Farmer", since Pulsar combines the logs of both Node and Farmer.